src/content/docs/release-notes/docs-release-notes/docs-29-05-2024.mdx

```diff
@@ -0,0 +1,39 @@
+---
+subject: "Docs"
+releaseDate: '2024-12-06'
+version: 'version: November 29-December 5, 2024'
+---
+
+### New docs
+
+* [Logging with the iOS agent](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ios/configuration/ios-remote-logging/) ensures consistent and comprehensive logging throughout the project, the iOS agent provides logging APIs. Use the agent's logging APIs to send your mobile apps logs to New Relic. 
+* [Introducing One-Step Kubernetes with APM auto-attach](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator) which was formerly known as the Kubernetes agent operator now streamlines full-stack observability for Kubernetes environments by automating APM instrumentation alongside Kubernetes agent deployment.
+
+### Major changes
+
+* Updated the version support information (under prerequisites section) about the [service broker for TAS](https://docs.newrelic.com/docs/infrastructure/other-infrastructure-integrations/cloudfoundry-integrations/vmware-tanzu-service-broker-integration/)
+* The synthetic job manager now supports Podman container environment. [Refer synthetic job manager to learn more about Podman.](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/)
```

Comment:
Refer [synthetic job manager](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/private-locations/install-job-manager/) to learn more about Podman.